### ISSUE TITLE
WebDAV browsing (and uploading?) support.

### ISSUE THREAD

**Issue by wyatt8740** [arpitkh96/AmazeFileManager#376]
I'd like to be able to browse files on a WebDAV server. SMB's already supported, and (apparently) SFTP's being worked on. But for me at least, WebDAV seems like a much simpler system to implement, and it is much easier to set up a WebDAV server in Apache, for example. It'd greatly simplify sharing files between my GNU/Linux desktop and my Android phone (especially if uploading files were implemented as well).

iFile on iOS supports this, and (I believe) some proprietary apps on the Google Play store do as well.

https://play.google.com/store/apps/details?id=com.rhmsoft.fm
https://play.google.com/store/apps/details?id=nextapp.fx

WebDAV is the one thing I am missing having since I started using Android on a nexus about three years ago. It'd make my day if we had a FOSS program that supported it.

**Comment by Montagnard65** [comment]
I agree with this enhancement! I'm mailbox.org user, and this provider allow Webdav to access files on their platform, but there isn't any open source app on F-droid which do that.
Do you think this enhancement could be soon added to this great app ?

**Comment by MrReSc** [comment]
I'm fully agree as well. I'm a mailbox.org user, too. This feature would be very helpful.

**Comment by vishal0071** [comment]
I'll definitely work on it, once we have a stable release. Stay tuned (:

**Comment by udopt** [comment]
I was wondering if there are any updates on the WebDAV support?

**Comment by kakolisgay** [comment]
+1 for this, Amaze should have WebDAV support by now.

**Comment by fin-ger** [comment]
Is it possible to donate for this feature?

**Comment by wyatt8740** [comment]
I'd still appreciate this as well. It's a really nice way to transfer files between my PC and phone without dealing with FTP server and port forwarding issues. Especially now that my phone's USB port is failing (it's really hard to get it to be detected from my PC), this would be appreciated.

**Comment by KOLANICH** [comment]
Transparent encryption should be implemented. I mean all the files uploaded are automatically encrypted and all the files downloaded are automatically decrypted. Though it will be hard to implement this right: if we upload a single block, an adversary will know which one is uploaded, if we upload multiple blocks if only one is updated, there will be big overhead.

**Comment by fin-ger** [comment]
@wyatt8740 I don't think encrypting the data is necessary for the initial feature. I am hosting a webdav server myself and have full control over my own data. What is necessary is a transport layer encryption to prevent sniffing the uploaded/downloaded data.

That's why I give a downvote for your suggestion.

**Comment by h-2** [comment]
Yes, simple support for webdav/webdavs would be great!

**Comment by lucker999** [comment]
Would be great to have this feature! 
Any progress on it?
